dap_aria_mapping/getters/taxonomies.py

```diff
@@ -1,7 +1,57 @@
-from nesta_ds_utils.loading_saving.S3 import download_obj
 from dap_aria_mapping import BUCKET_NAME
+from nesta_ds_utils.loading_saving.S3 import download_obj
+import boto3, yaml, pickle
 import pandas as pd
-import yaml
+from toolz import pipe
+
+
+def get_embeddings() -> pd.DataFrame:
```

Comment:
embeddings of the entities? would be helpful to specify in docstring (vs abstracts, etc.)